lib/riemann/tools.rb

```diff
@@ -65,6 +65,7 @@ def report(event)
       end
       
       event = event.merge(attributes)
+      event[:time] = Integer(event[:time]) if event.has_key? :time
```

Comment:
(Would reraise, of course. Just don't want others to have to waste a day figuring it out again.)